Source/App/HeadlessMain.cpp

```
@@ -253,6 +255,66 @@ static AudioGraph buildLoopGraph(const std::shared_ptr<AudioBuffer>& source,
     return graph;
 }
 
+static std::shared_ptr<TrackManager> makeTrackManager(uint32_t sampleRate) {
+    auto trackManager = std::make_shared<TrackManager>();
+    trackManager->setOutputSampleRate(static_cast<double>(sampleRate));
+    trackManager->setInputSampleRate(static_cast<double>(sampleRate));
+    trackManager->setInputChannelCount(0);
+    return trackManager;
+}
+
+static std::string makeRoundTripPath(const std::string& name) {
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)